frontend/src/app/containers/App.js

```diff
@@ -74,7 +80,19 @@ class App extends Component {
     }
   }
 
-  componentDidUpdate() {
+  componentDidMount() {
```

Comment:
We have to do a lot more stuff in App.componentDidMount, since we have to wait until here to be able to access anything on the window object.